ci/scripts/update-credhub-db.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Does this script need to exist? If flyway expects an empty database, do we have to do any of this?

Reply:
The `create-and-update.sh` script adds some extensions.  When I did this work, I assumed those extensions were needed for compliance.  If that's not the case it would be _much_ easier to pull this script.

Reply:
I don't know of a compliance need here. cc @LinuxBozo

Reply:
No, there is no compliance need, there was however various other needs for either BOSH or CF so we just added them to both, but I don't know that Credhub has the same needs, though, it might when you know what each of these does.

citext: tolower string comparison in SQL queries
uuid-cissp: exactly how it sounds, for generating uuid
pgcrypto: for doing password hashing or other digests, like sha or md5
pg_stat_statements: keeping running stats of all queries that run

In reality, I have a feeling that pg_stat_statements is the one that is causing Credhub issues. So yeah, I would start with none, and then only add any that Credhub requires.

Reply:
Yeah that was what our plan was for this.  From some research it seems that only `pg_stat_statements` and `citext` need some kind of state in the DB.  I was surprised by the `citext` needed some storage but from the docs it seems to be it needs storage for the operators.